lib/tasks/courses.rake

```diff
@@ -0,0 +1,41 @@
+namespace :courses do
+  desc 'Seed default courses'
+  task seed: :environment do
+    def create_courses!(category, weekdays, slots)
+      weekdays.to_a.foreach do |weekday_name, weekday_index|
+        slots.foreach do |slot|
```

Comment:
The object method `foreach` does not exist. Can you please replace it by `each`?